### ISSUE TITLE
How many epochs to run on VBD dataset to reproduce the results in the sgmse+ paper?

### ISSUE THREAD

**Issue by Yangliusha** [sp-uhh/sgmse#57]
Thank you for the open source of this awesome work. I'm trying to reproduce the results in the paper. But after 870 epochs of training (far more than the 160 epochs written in the paper), I only got PESQ: 1.95+-0.69. Is there anything I'm missing to run the codes? I run the codes with all the default settings.

**Comment by HarperShi** [comment]
Same problem here. I have trained this model with vctk_28spk (12k samples) before and it's alright able to generate quite decent output. But somehow when I tried to use the whole vctk (35k samples ) corpus, it' not able to get converged anymore. Instead, it will always get stuck at around PESQ=1.5

**Comment by libai2345** [comment]
Hi, have you solved this problem now?

**Comment by Huanianss** [comment]
> Thank you for the open source of this awesome work. I'm trying to reproduce the results in the paper. But after 870 epochs of training (far more than the 160 epochs written in the paper), I only got PESQ: 1.95+-0.69. Is there anything I'm missing to run the codes? I run the codes with all the default settings.

I trained the model with a batch size of 8 and a gradient accumulate step of 4 to approximate the batch of 32 on the 4 GPUs described in the paper.  Other settings are the default and the PESQ: 2.76 ± 0.66.  May I ask if have you tested with the pre-trained weight? I tested the pretrained weight on the VB-DMD dataset and the PESQ is 2.91 ± 0.63.

**Comment by julius-richter** [comment]
Different batch sizes, GPUs, or CUDA architectures can lead to slight deviations in the final results. We also verified that the pretrained checkpoint reproduces the reported performance reasonably well.

I will close this issue due to inactivity.
